app/internal/friend_view.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from sqlalchemy.orm import Session
+
+from app.routers.event import sort_by_date
+from app.routers.user import get_all_user_events
+
+
+def get_events_per_friend(
+        session: Session,
+        user_id: int,
+        my_friend: str,
```

Comment:
please add documentation.
also my friends is a string by what column? name? username?
I suggest that it will be something that is unique.

Reply:
I added this documentation:
My_friend is the name of a person that appears in the invite list events.
He is not necessarily a registered user.
The variable, my_friend, is used to show all the events where we are both on the invitees' list